README.md

```diff
@@ -141,11 +141,11 @@ model:
       acceptance:
         - scenario: onReceive
           given:
-          - The EchoService is running.
+           - The EchoService is running.
```

Comment:
Should the spaces here be the same?  Non-blocking, not a big deal.

Reply:
Good catch, I'll double check. Copy/Paste may have dorkered.

Reply:
No, this is the corrected spacing.

Reply:
The example model given previously was improperly formatted in a few places, that was what this was trying to address.